### PR TITLE
chore(readme): update readme to contain info for setting up a proxy w…

### PR DESCRIPTION
**Pull request by kevinkowa**
update readme to contain info for setting up a proxy with the authenticator

### CODE REVIEW

File: assistant/src/test/java/com/ibm/watson/assistant/v1/AssistantServiceIT.java [hunk 1]
```diff
@@ -1770,7 +1770,7 @@ public void testDeleteUserData() {
   /** Test list mentions. */
   @Test
   public void testListMentions() {
-    String entity = "holiday";
```
Comment (apaparazzi0329) [inline assistant/src/test/java/com/ibm/watson/assistant/v1/AssistantServiceIT.java:1773]: Why the change from holiday to beverage?

Comment (kevinkowa) [reply assistant/src/test/java/com/ibm/watson/assistant/v1/AssistantServiceIT.java:1773]: it was failing, I'm not sure if something got deleted in the assistant

**Review by apaparazzi0329 — APPROVED** [review]
LGTM
